CONTRIBUTING.md

```diff
@@ -1,28 +1,71 @@
+
+### Installation
+
+- Run `yarn` in the repository's root directory to install everything you need for development.
+- Run `yarn bootstrap` in the root directory to build and set up the packages.
+
+### Available Scripts
+
+All packages related to the design system are organized in this monorepo. Thanks to [`yarn workspaces`](https://classic.yarnpkg.com/en/docs/workspaces) packages that depend on each other always use the latest version. [`lerna`](https://lerna.js.org/) makes it possible to run scripts across all packages at the same time. The list of scripts below can be run in each package directory individually, or in the repository's root directory for all packages at once.
+
+- `yarn start` — will build packages on every change
+- `yarn build` — will build packages once
+- `yarn test` — will run unit tests on every change
+- `yarn lint` — will lint the code once
+- `yarn docs` — will run the [Storybook](https://storybook.js.org/) in development mode
+
+Refer to the `package.json` files in each package for other helpful scripts.
+
+### Changesets
+
+Circuit UI uses [changesets](https://github.com/atlassian/changesets) to do versioning. As a contributors you can add a changeset by running `yarn changeset`. Read more in our [release process docs](https://circuit.sumup.com/?path=/docs/introduction-contributing-release-process--page).
+
+### Troubleshooting
+
+**Changes in one package aren't recognized in another package**
+
+You need to rebuild a package after making changes to it. The easiest way to do so is to run `yarn start` in the repository's root directory which will continuously rebuild all packages on change.
 
-### Submitting an issue
+If the changes aren't picked up even after a rebuild, try restarting or reloading your editor.
 
-1. Check existing issues and verify that your issue is not already submitted. If a similar issue already exists, we highly recommend to add your report to that issue.
-2. Open issue using one of the templates.
-3. Be as detailed as possible - include the package version, what you did, what you expected to happen, and what actually happened.
+**An error is thrown when creating a changeset**
 
-### Submitting a PR
+`Error: Failed to find where HEAD diverged from main. Does main exist?`
 
-1. Find an existing issue to work on or follow `Submitting an issue` to create one that you're also going to fix. Make sure to notify that you're working on a fix for the issue you picked.
-2. Branch out from the latest `main` branch.
-3. Code, add, commit and push your changes in your feature branch.
-4. Make sure that tests and linter(s) pass locally for you.
-5. Submit a PR.
-6. Collaborate with the codeowners/reviewers to merge your changes to `main`.
+This error occurs when the `main` branch doesn't exist locally. Check out the `main` branch (`git checkout -b main`), pull the latest changes (`git pull`), then check out your feature branch (`git checkout -`) and try again.
```

Comment:
This is really useful in the process of the migration to a monorepo (happened to me and Amela, will happen to others) but in the future we can probably remove it, since we're asking people to branch off of `main` anyways 🙂